packages/cosmos_ui_components/lib/components/mnemonic_choice_chip.dart

```diff
@@ -17,42 +18,54 @@ class MnemonicChoiceChip extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final textTheme = Theme.of(context).textTheme;
-    const lightInactive = CosmosColorsData.lightInactive;
-    const lightDivider = CosmosColorsData.lightDivider;
     return InkWell(
       onTap: onTap,
       child: Container(
         decoration: BoxDecoration(
-          borderRadius: BorderRadius.all(Radius.circular(CosmosTheme.of(context).radiusS)),
-          border: Border.all(
-            color: lightInactive.withOpacity(word.isEmpty ? lightInactive.opacity / 2 : lightInactive.opacity),
-          ),
-          color: lightDivider.withOpacity(word.isEmpty ? lightDivider.opacity / 2 : lightDivider.opacity),
+          borderRadius: BorderRadius.all(Radius.circular(CosmosTheme.of(context).radiusM)),
+          color: CosmosTheme.of(context).colors.chipBackground,
         ),
         child: Padding(
-          padding: EdgeInsets.symmetric(
-            horizontal: CosmosTheme.of(context).spacingS,
-            vertical: CosmosTheme.of(context).spacingXS,
+          padding: EdgeInsets.only(
+            left: CosmosTheme.of(context).spacingM,
+            right: CosmosTheme.of(context).spacingL,
+            top: CosmosTheme.of(context).spacingM,
+            bottom: CosmosTheme.of(context).spacingM,
           ),
           child: Row(
             mainAxisSize: MainAxisSize.min,
             children: [
               if (showIndex) ...[
-                Text(
-                  "$index",
-                  style: textTheme.bodyText1,
-                ),
-                SizedBox(width: CosmosTheme.of(context).spacingS),
+                _indexText(context, index),
               ],
               Text(
                 word,
-                style: textTheme.bodyText1?.copyWith(fontWeight: FontWeight.w700),
+                style: CosmosTextTheme.labelS,
               ),
             ],
           ),
         ),
       ),
     );
   }
+
+  Widget _indexText(BuildContext context, int index) => Padding(
+        padding: EdgeInsets.only(right: CosmosTheme.of(context).spacingM),
+        child: Container(
+          width: 18,
+          height: 18,
```

Comment:
Do we have a special requirement of an 18x18 text box?

Reply:
its according to the figma design:
<img width="205" alt="image" src="https://user-images.githubusercontent.com/1711474/143866669-9b263a48-0aa9-4b41-a27b-232fefe477dd.png">

Reply:
Perfect